### DISCUSSION TITLE
Spend on One-pager and spend on Budget Allocator do not agree

### DISCUSSION THREAD

**Discussion by Snowcatcat** [facebookexperimental/Robyn#335] [Q&A]
Hi,

I chose the model and used it to run the budget allocator, but I found that the spend shares are different. Is there a way to force the budget allocator to look at the modeling period instead of all historical data inclusive?

![image](https://user-images.githubusercontent.com/99045100/156962476-3540203c-a98c-4fa9-8c63-3ad793859ce3.png)

![image](https://user-images.githubusercontent.com/99045100/156962530-3d9d7a05-f81d-4cda-ad42-52bf3d45259c.png)

**Comment by kyletgoldberg** [comment]
Hey there - check this issue for a discussion on this topic. https://github.com/facebookexperimental/Robyn/issues/227?fbclid=IwAR2BuTcmqb6pNw-DEznwI9TmPBMd8MYWI02EvllKamBldeJQOVMtGtcZIR4
